LIPS/lip-3.md

```diff
@@ -0,0 +1,160 @@
+| lip: | title: | status: | author: | discussions-to: | created: | updated: |
+| --- | --- | --- | --- | --- | --- | --- |
+| 3 | Easy Track Motions for DAO routine operations | WIP | Bogdan Kovtun (psirex), Gregory Stepanov (grstepanov) | https://research.lido.fi/t/lip-3-easy-track-motions-v2/794 | 2021-07-09 | |
+
+# Summary
+
+## Problem
+
+Lido DAO governance currently relies on Aragon Voting App model. This means DAO approves or rejects proposals via direct governance token voting. Though transparent and reliable, it is not a convenient way to make decisions only affecting small groups of Lido DAO members. Besides, direct token voting doesn't exactly reflect all the decision-making processes within the Lido DAO and is often used only to formalize an existing consensus.
+There are a few natural sub-governance groups within the DAO, e.g. node operators committee, financial operations team and LEGO committee. Every day they need to take routine actions only related to their field of expertise. The decisions they make hardly ever spark any debate in the community, and votings on such decisions often struggle to get the quorum and get enacted on-chain.
+
+## Proposed solution
+
+We propose Easy Track frictionless motions as a solution to this problem. 
+Easy Track motion is a lightweight voting considered to have passed if the minimum objections threshold hasn’t been exceeded within the preset voting duration timeframe. As opposed to traditional Aragon votings, Easy Track motions are cheaper (no need to vote ‘pro’, token holders only have to vote ‘contra’ if they have objections) and easier to manage (no need to ask broad DAO community vote on proposals sparking no debate). 
+
+## Use cases for Easy Track motions
+
+There are three types of votings run periodically by the Lido DAO that are proposed to be wrapped into the new Easy Track motions:
+- Node Operators increasing staking limits
+- Funds being allocated into reward programs
+- Funds being allocated to LEGO program
+
+More types of motions can be added later, e.g.:
+- Topping up Lido staking referral program
+- Purchasing staking cover
+
+## Sub-governance groups
+
+There's a formal or an informal sub-governance group within the Lido DAO for each of the voting cases listed above. Those groups have been formed intentionally or in a more natural way, and at this point creating and maintaining periodical Aragon votings falls within their remits.
```

Comment:
For the last part, it's not creating the votings, but ensuring the necessary transactions get processed through means of Aragon vote. The groups don't start the votes in general, and only manage actions required for their respective ops

Reply:
Fixed.